Graphs/AbstractGraph.js

```diff
@@ -43,12 +43,12 @@ export default class AbstractGraph extends React.Component {
     componentDidUpdate(prevProps) {
         if (!_.isEqual(prevProps.configuration.data, this.props.configuration.data)) {
             this.setConfiguredProperties(this.props, this.properties);
-            this.reInitializeTooltip(this.props);
+            this.reInitializeTooltip(prevProps);
```

Comment:
prevProps cannot be used to reinitialize in componentDidUpdate

Reply:
reInitializeTooltip() was comparing this.props.configuration.data against the same this.props.configuration.data, and so was never invoking this.setTooltip()
setTooltip() will anyway use latest props.

what other fix would you suggest?

Reply:
I see! This function is redundant then please remove it and bring the code into componentDidUpdate